src/live_restore/live_restore_fs.c

```diff
@@ -934,18 +929,15 @@ __wti_live_restore_cleanup_tombstones(WT_SESSION_IMPL *session)
     fs->finished = true;
 
     WT_RET(__wt_scr_alloc(session, 0, &filepath));
-    /*
```

Comment:
This fixme made this ticket not a quick win, can create another ticket to do this. @ajmorton  what do you think?

Reply:
Yep happy to bump this. You mentioned in the other PR that folders like `diagnostic.data` will be a problem so this could turn into a hairy problem.